contrib/ansible/vagrant/openstack_config.yml

```diff
@@ -0,0 +1,12 @@
+os_username: eparis
```

Comment:
So I wanted to put real(ish) data here as an example.  Also, I thought the .gitignore trick worked, but it doesn't...

Another option would be to make this file be .gitignore and make a openstack_config.yml.example. And the end user would have to cp the .example to just .yml and then make edits....

I'd still probably leave this data for reference though...